### ISSUE TITLE
Support for source maps

### ISSUE THREAD

**Issue by avalez** [Tug/express-asset-manager#5]
Hi Tug,

Could you please consider adding support for source maps?

Without making it configurable and strictly for gcc it's that simple:

    diff --git a/lib/AssetGroupProcessors.js b/lib/AssetGroupProcessors.js
    index 9543d0d..a2962ee 100644
    --- a/lib/AssetGroupProcessors.js
    +++ b/lib/AssetGroupProcessors.js
    @@ -124,7 +124,8 @@ module.exports = function(options) {
                             fileIn      : assetGroup.assetsPaths(),
                             fileOut     : assetGroup.filepath,
                             buffer      : options.buffer,
    -                        callback    : callback
    +                        callback    : callback,
    +                        options     : ["--create_source_map", assetGroup.filepa
                         });
                         this.default(assetGroup);
                     } else {

Thank you!

**Comment by Tug** [comment]
True, simple js assets do not accept node-minify's `options` parameter.
Note that it works already with `require-js` assets using the `generateSourceMaps` option.
This feature seems easy enough, I might add it soon, thanks!

**Comment by Tug** [comment]
Oh, I didn't see your pull request, reviewing it now :)

**Comment by avalez** [comment]
Thanks! I did not notice generateSourceMaps option through code. Also, I just have noticed that you have fixed tests in branch. Please let me know if you'd like me to adjust my pull request to exclude 'fix tests' change.

**Comment by Tug** [comment]
generateSourceMaps is a requirejs option only. You can either set it for all assets in manager's config:
```
var assetManagerConfig = {
    rootRoute   : "/static",
    srcDir      : "./public",
    buildDir    : "./builtAssets",
    process     : "true",
    requirejs: {
        optimize: "none",
        preserveLicenseComments: true,
        generateSourceMaps: true
    }
};
```
Or in your require.js module config file (the one given for mainConfigFile).

I didn't see any needs to adjust your pull request, it looked fine to me.

**Comment by avalez** [comment]
Nice! May be my pull request is worth adjusting to support same syntax?

**Comment by avalez** [comment]
Also will you please publish an update?

**Comment by Tug** [comment]
I have just published v0.3.0, which includes the generateSourceMaps option for node-minify (see test).
